test_spec.py

```diff
@@ -466,6 +466,22 @@ def test_namedtuple_data(self):
 
         self.assertEqual(result, expected)
 
+    def test_get_key_not_in_dunder_dict_returns_attribute(self):
+        class C:
+            foo = "bar"
+
+        instance = C()
+        self.assertNotIn("foo", instance.__dict__)
```

Comment:
`assertNotIn` is new to python3.1. I know python2 is sunset and everything so I have no right to be suggesting compatibility with it. But it is an odd sense of pride and the reason I made this library in the first place. 

```suggestion
        self.assertTrue("foo" not in instance.__dict__)
```